bin/sl-runctl.js

```diff
@@ -189,14 +189,14 @@ function requestHeapSnapshot() {
   var name = optionalArg(util.format('node.%s', target));
   request.cmd = 'heap-snapshot';
   request.target = target;
-  // .heapdump extention Required by Chrome
-  request.filePath = path.resolve(name + '.heapdump');
+  // .heapdump extention required by Chrome
```

Comment:
@sam-github note the comment you case-corrected. Looks like a decent reason to standardize on `.heapdump` instead of `.heapsnapshot`.